src/components/feedback/__tests__/feedback-sent-rating.test.js

```diff
@@ -60,7 +67,13 @@ describe('Sent helpful rating - no', () => {
         pathname: '/mapbox-gl-js/api/',
         hash: '#lnglat'
       }}
-      userName="decorah"
+      user={{
+        id: 'decorah',
+        email: 'decorah@mapbox.com',
+        plan: {
+          id: 'staff'
```

Comment:
Is there anything bad that could happen by putting your username, email, and the fact that it's a staff account into a public codebase? I'm not sure if there's any actual risk here, but I thought it would be worth raising the question.

Reply:
Ohhh good point. I'll put in some dummy data instead!